### PR TITLE
Remove Redundant Gift Card Documentation

### PR DESCRIPTION
**Pull request by AndyClifford**
Epay Gift Cards are no longer a Centrapay managed asset. We had a lot of documentation revolving around the loading and sending of gift cards into and within our system as well as references to the old epay.nzd asset type. These have been updated to use the new asset type and deprecated API responses have been updated with token examples which are still valid.

**Ticket:** https://www.notion.so/centrapay/Epay-Migration-0e1ecee27a9644f992ae5054ae114bd1?pvs=4

**Test Plan:**
- [ ] Navigate to each of the pages that have been modified in this PR.
- [ ] Check the documentation is as expected

### CODE REVIEW

File: src/content/guides/partial-payment-extension.mdx [hunk 1]
```diff
@@ -46,7 +46,7 @@ autonumber
 A patron asks the merchant to process a specified part of the total basket
 amount using Centrapay. This could be due to the patron knowing their Centrapay
 Asset balance wouldn’t cover the complete basket amount or enable them to use a
-set amount of funds from a fixed value asset eg. Gift Card to complete the payment.
```
Comment (alexBinns) [inline src/content/guides/partial-payment-extension.mdx:49]: Can still pay with giftcard I wonder how we should word this or token is ok?

File: src/pages/index.astro [hunk 2]
```diff
@@ -49,7 +49,7 @@ const img = '/homepage-preview.png';
       </h1>
       <p class="type-body-1 max-w-[720px]">
         Welcome to Centrapay! We enable you to transact fiat currency,
-        giftcards, digital assets and vouchers via your point of sale,
```
[the remark this answers is not in the document]
Comment (AndyClifford) [reply src/pages/index.astro:52]: digital assets probably covers gift cards too? I'm happy either way

File: src/content/api/assets.mdx [hunk 3]
```diff
@@ -17,7 +17,7 @@ Centrapay account to perform transactions where value is exchanged. Assets can
 be spent to satisfy [Payment Requests](/api/payment-requests), withdrawn to a bank account via
 [Funds Transfers](/api/funds-transfers) and sent to other Centrapay users via [Asset Transfers](/api/asset-transfers).
 
-Assets are categorized as either Money, Gift Cards or Tokens. Depending on its
+Assets are categorized as either Money or Tokens. Depending on its
```
[the remark this answers is not in the document]
Comment (AndyClifford) [reply src/content/api/assets.mdx:20]: Why?

File: src/content/api/asset-types.mdx [hunk 4]
```diff
@@ -11,19 +11,19 @@ The following table describes the Asset Types supported for payments.
...
+|   Asset Type    |                     Description                     |             Category             |          Currencies           |  Flags   |
+| :-------------- | :-------------------------------------------------- | :------------------------------- | :---------------------------- | :------- |
+| bitcoin         | [Bitcoin](https://bitcoin.org/en/)                  |                                  | `NZD` `AUD`                   | ⚡️       |
+| cca.coke        | Coke tokens                                         | [Token](/api/assets#token-model) | `NZD`                         | ⚡️ 🧪 🚫   |
+| centrapay.nzd   | Centrapay NZD wallet                                | [Money](/api/assets#money-model) | `NZD`                         | ⚡️ 🧪     |
+| centrapay.token | Centrapay tokens                                    | [Token](/api/assets#token-model) | `NZD`                         | ⚡️ 🧪     |
+| epay            | [EPay NZ giftcards](https://www.giftstation.co.nz/) |                                  |                               | ⚡️ 🧪     |
```
Comment (alexBinns) [inline src/content/api/asset-types.mdx:20]: I think we still need to mention the currency

File: src/content/guides/partial-payment-extension.mdx (hunk 1 above)
Comment (AndyClifford) [reply src/content/guides/partial-payment-extension.mdx:49]: I'm easy, it's just an example so Token is fine with me. We still have a couple of references to gift cards elsewhere

File: src/content/api/_endpoints/merchants-search.js [hunk 5]
```diff
@@ -24,15 +24,7 @@ export default {
...
-            assetType: 'epay.nzd.main',
-            products: [
-              {
-                id: '37873'
-              },
-              {
-                id: '38183'
-              }
-            ]
+            assetType: 'epay.main',
```
Comment (MeganSteenkamp) [inline src/content/api/_endpoints/merchants-search.js:27]: Would they still have the product id in this example?

**Review by MeganSteenkamp — APPROVED** [review]

File: src/content/api/_endpoints/merchant-configs-create.js [hunk 6]
```diff
@@ -13,7 +13,7 @@ export default {
           walletId: '1234c486308f3f0a23f0f92b'
         },
         {
-          type: 'epay.nzd.main',
+          type: 'epay.main',
```
Comment (Ray2126) [inline src/content/api/_endpoints/merchant-configs-create.js:16]: Looks like `epay.main` asset type requires `externalMerchantId`, not `terminalId`, based on the code.

File: src/content/guides/loading-and-sending-assets.mdx [hunk 7]
```diff
@@ -78,125 +78,3 @@ Clicking on the link will open a page to view and claim the asset transfer.
   }
   ```
   </CodePanel>
-
-## Loading Giftcards
-
-You can load Giftcards by calling our [External Assets](/api/external-assets) endpoint. You will need to use the giftcard number for the `externalId` field. The `pin`, the `issuer` and the `type` need to be on hand too.
```
Comment (Ray2126) [inline src/content/guides/loading-and-sending-assets.mdx:84]: What are we doing with this load external asset endpoint?

**Review by Ray2126 — APPROVED** [review]

File: src/content/api/_endpoints/merchant-configs-create.js (hunk 6 above)
Comment (alexBinns) [reply src/content/api/_endpoints/merchant-configs-create.js:16]: You are right good catch

File: src/content/guides/loading-and-sending-assets.mdx (hunk 7 above)
Comment (alexBinns) [reply src/content/guides/loading-and-sending-assets.mdx:84]: Tom said to archive it for now as no one is using it and we may want to change it before someone else uses it as it is quite giftcard specific. See thread here https://centrapay.slack.com/archives/C03HXJDPPB4/p1713390769161409

File: src/content/api/assets.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (alexBinns) [reply src/content/api/assets.mdx:20]: NVM

**Pull request merged**